crates/storage-pg/migrations/20250924132713_personal_access_tokens.sql

```diff
@@ -0,0 +1,55 @@
+-- Copyright 2025 New Vector Ltd.
+--
+-- SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+-- Please see LICENSE in the repository root for full details.
+
+-- A family of personal access tokens. This is a long-lived wrapper around the personal access tokens
+-- themselves, allowing tokens to be regenerated whilst still retaining a persistent identifier for them.
+CREATE TABLE personal_sessions (
+    personal_session_id UUID NOT NULL PRIMARY KEY,
+    owner_user_id UUID NOT NULL REFERENCES users(user_id),
```

Comment:
I wonder if this could be nullable because it could be possible to create a personal session using the client credentials? In this case the owner is the client I guess?